src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -172,13 +172,6 @@ public static class MsalError
         /// </summary>
         public const string InvalidOwnerWindowType = "invalid_owner_window_type";
 
-        // TODO: InvalidServiceUrl does not seem to be used anywhere?
-
-        /// <summary>
-        /// Invalid service URL.
-        /// </summary>
-        public const string InvalidServiceUrl = "invalid_service_url";
```

Comment:
>public const string InvalidServiceUrl = "invalid_service_url"; [](start = 8, length = 62)

Like the cleanup, however perhaps use obsolete instead, it's a breaking change to remove them if used or not?  (comment relevant in a few places)